controllers/projects.js

```diff
@@ -23,12 +31,63 @@ router.get('/new', function(req, res) {
   res.render('projects/new');
 });
 
+// POST from form action/projects/edit/<project.id>
+router.post("/edit/:id", function(req, res) {
+  // console.log(req.body);
+  db.project.update({
```

Comment:
re: your question about the edit not deleting the old category. here you update the details which is great .. see below for more